### PR TITLE
IMeshTag about (N, 1) shaped array

### PR DESCRIPTION
**Pull request by zxkjack123**
Closed #969 by mistake. This is the identical one as that.
Rebased on develop.

### CODE REVIEW

**Review by scopatz — APPROVED** [review]

**Comment by scopatz** [comment]
Also, please add a news entry to this. We don't have documentation for this yet, but it is effectively the same as following the instructions here: http://xon.sh/devguide.html#changelog

**Review by pshriwise — APPROVED** [review]
This looks good. Thanks @zxkjack123

File: pyne/mesh.py [hunk 1]
```diff
@@ -412,17 +412,21 @@ def __setitem__(self, key, value):
             mtag[i_ve[1]] = value if tsize == 1 else value[0]
         elif isinstance(key, slice):
             key = list(miter)[key]
-            v = np.empty(len(key), self.tag.type) if tsize == 1 else \
-                np.empty((len(key), tsize), self.tag.type)
+            if tsize == 1 and len(value.shape) == 1:
```
Comment (gonuke) [inline pyne/mesh.py]: Could you use the same trick as in #965 and just reshape this if the size is 1?
```
v = np.empty((len(key), tsize), self.tag.type)
if  tsize == 1 and len(value.shape) == 1:
    v.shape = (len(key), )
```

**Review by gonuke — COMMENTED** [review]
Looking for less code duplication and consistency with other places where we do this.

File: pyne/mesh.py (hunk 1 above)
Comment (zxkjack123) [reply pyne/mesh.py]: Codes are changed here according to your suggestion. @gonuke

**Comment by zxkjack123** [comment]
@scopatz A news was added to this PR.

**Comment by gonuke** [comment]
Thanks @zxkjack123

**Pull request merged**